android-lib/build.gradle

```diff
@@ -124,7 +124,7 @@ dependencies {
   classpaths files(new File(System.getenv('ANDROID_HOME') + '/platforms/android-19/android.jar'))
   classpaths files('build/classes/release')
   //testing
-  androidTestCompile 'com.android.support:multidex-instrumentation:1.0.3'
```

Comment:
Got compilation errors saying the version does not match, so I had to downgrade to a early version